viewing_party/party.py

```diff
@@ -1,23 +1,217 @@
+import statistics 
+from statistics import mode 
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    """
+    input: title, genre, rating
+    output: dictionary with the three keys "title", "genre", and "rating"
+    """
+    movie_dict = {}
+    if title and genre and rating:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+
+    return None 

```

Comment:
👍 Great use of a docstring! We can use a guard clause to exit the function early and return the dictionary literal itself to save 1 line (a really minor nitpick). 

```python
def create_movie(title, genre, rating):
    # Input validation
    if not title or not genre or not rating:
        return None

    return {
        "title": title,
        "genre": genre,
        "rating": rating
    }
```